src/components/Sidebar/Sidebar.stories.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Meta, StoryFn } from '@storybook/react';
+import { withDesign } from 'storybook-addon-designs';
+import { Sidebar, SidebarProps } from './Sidebar';
+import { IconButton } from '../IconButton';
+import { MdClose } from 'react-icons/md';
+import styled from 'styled-components';
+import { pxToRem } from '../../shared';
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
```

Comment:
Pistä tähän argsit kuntoon oletusarvojen mukaan, jotta käyttö storybookissa olisi kätevämpää. Eli propertyissä ei saisi näkyä näitä set boolean nappuloita. 
![image](https://github.com/cybercom-finland/knowit-finland-design-system/assets/15112008/66241d17-6516-4ed2-b543-e074192fd825)